cluster/ci/config/prow/config.yaml

```diff
@@ -538,7 +538,6 @@ branch-protection:
           required_status_checks:
             contexts:
             - Build
-            - GUI-Tests
             - jenkins/job
             - Performance
```

Comment:
You should remove `Performance` and `GUI-Tests-New-App` below since they've moved to prow.